utils/config.go

```diff
@@ -13,7 +13,10 @@ var RaspiFamily = resource.NewModelFamily("viam", "raspberry-pi")
 
 // BoardSettings contains board-level configuration options.
 type BoardSettings struct {
-	TurnI2COn bool `json:"turn_i2c_on,omitempty"`
+	TurnI2COn    bool  `json:"turn_i2c_on,omitempty"`
+	BTenableuart *bool `json:"bluetooth_enable_uart,omitempty"`
```

Comment:
why pointers? i haven't seen pointer usage for primitives for deserializing json. does this even work!?